.travis.yml

```diff
@@ -34,6 +34,9 @@ jobs:
     env:
     - CHANGE_MINIKUBE_NONE_USER=false
     - MINIKUBE_HOME=${HOME}
+    - TEST_PROVISION_TIMEOUT=5m
```

Comment:
We intentionally don't provide `TEST_BROKER_READY_TIMEOUT` right? Might be a good idea to have a comment about that.